lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -324,6 +324,11 @@ function dosomething_reportback_form_validate($form, &$form_state) {
 function dosomething_reportback_form_submit($form, &$form_state) {
   $values = $form_state['values'];
 
+  $campaign = node_load($values['nid']);
+  global $user;
+  $language = dosomething_global_get_language($user, $campaign);
```

Comment:
It's better to name this `$language_code` to avoid confusion, as `$language` is global variable.